db/python/tables/bq/billing_ar_batch.py

```diff
@@ -0,0 +1,69 @@
+from datetime import datetime, timedelta
+
+from google.cloud import bigquery
+
+from api.settings import BQ_BATCHES_VIEW
+from db.python.tables.bq.billing_base import BillingBaseTable
+
+
+class BillingArBatchTable(BillingBaseTable):
+    """Billing AR - BatchID lookup Big Query table"""
+
+    table_name = BQ_BATCHES_VIEW
+
+    def get_table_name(self):
+        """Get table name"""
+        return self.table_name
+
+    async def get_batches_by_ar_guid(
+        self, ar_guid: str
+    ) -> tuple[datetime, datetime, list[str]]:
+        """
+        Get batches for given ar_guid
+        """
+        _query = f"""
+        SELECT
+            batch_id,
+            MIN(min_day) as start_day,
+            MAX(max_day) as end_day
+        FROM `{self.table_name}`
+        WHERE ar_guid = @ar_guid
+        AND batch_id IS NOT NULL
+        GROUP BY batch_id
+        ORDER BY 1;
```

Comment:
Does ORDER BY 1 mean order by the `batch_id`, can we use that alias instead?

Reply:
Yes, good point, let's use batch_id here as we already have it in GROUP BY